clients/admin-ui/src/features/system/SystemInformationForm.tsx

```diff
@@ -153,7 +161,24 @@ const SystemInformationForm = ({
     values: FormValues,
     formikHelpers: FormikHelpers<FormValues>
   ) => {
-    const systemBody = transformFormValuesToSystem(values);
+    let dictionaryDeclarations;
+    if (lockedForGVL) {
+      const dataUseQueryResult = await getDictionaryDataUseTrigger({
+        vendor_id: values.vendor_id!,
+      }).unwrap();
```

Comment:
we should avoid using `unwrap`—I know we have it a few places in the code base, but it's a pattern we are trying to get away from. see this old convo for why: https://github.com/ethyca/fides/pull/1489#discussion_r1003890806

Reply:
Okay-- for my error handling, I think if the dictionary data use query fails here, probably the whole submit should fail, what would be the cleanest way to error out of it here?